Mathlib/FieldTheory/RatFunc.lean

```diff
@@ -598,6 +604,7 @@ instance instCommRing : CommRing (RatFunc R) :=
 
 section LiftHom
 
+variable {K}
```

Comment:
The mathport output has `variable {K}` _before_ `section LiftHom`.

Reply:
Should I then move the `variable {K}` where it was and also reinstate (if needed) `variable (K)` after the `end LiftHom`?

Reply:
You should aim to match the mathport output where possible; so yes, reinstate the variables lines you deleted and moved.

Reply:
Ok, I re-instated the previous variable update and hence added back the re-update!